stable/insights-admission/templates/configuration.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
when was v1 introduced? We might want to continue supporting the old version - I think we still have some folks as far back as 1.14

Reply:
Check out `Capabilities.Has` for how to do this - we have it in a few other places in this chart

Reply:
@rbren  While looking into this, the current master version of the chart does not work with Kube 1.14 - to me that means we don't need to support 1.14?

insights-agent install output on Kube 1.14 from chart master:
```
Release "insights-admission" does not exist. Installing it now.
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(ValidatingWebhookConfiguration.webhooks[0].clientConfig.service): unknown field "port" in io.k8s.api.admissionregistration.v1beta1.ServiceReference, ValidationError(ValidatingWebhookConfiguration.webhooks[0]): unknown field "matchPolicy" in io.k8s.api.admissionregistration.v1beta1.Webhook, ValidationError(ValidatingWebhookConfiguration.webhooks[0]): unknown field "objectSelector" in io.k8s.api.admissionregistration.v1beta1.Webhook]
```

Reply:
Update, we can support Kube 1.15 (which does currently work with our master branch). I've updated the template with if/else statements for the API version, and tested in Kube 1.15 and again in 1.22.